### DISCUSSION TITLE
About value of the loss and of the absorption

### DISCUSSION THREAD

**Discussion by JessicaFerrari89** [Ifsttar/I-Simpa#319] [Q&A]
Hi, during a simulation of tutorial 2 I tried to change some properties in the materials: I check "Transmission" and used the following values (for example to 125Hz): 0.45 for Absorption, 0.30 for Diffusion, 3.5 for Loss with Uniform as the type of Diffusion. But I received the following message "Warning: The value of the loss is not corresponding to the value of the absorption.The value of loss has been fixed to 3 dB, and the absorption value keep to 0.45".
I tried to change values but without success.
What I wrong?
Thank you.

**Comment by nicolas-f** [comment]
Hi,

There is a formula that constrain the values between theses columns, you can find more details in the documentation:

https://i-simpa-wiki.readthedocs.io/fr/latest/project_database.html#properties

![image](https://user-images.githubusercontent.com/1382241/175881875-dc7e94ba-adcc-411a-abdd-e6efda9dbcee.png)

**Reply by JessicaFerrari89** [reply]
Ok thank you so much for your explanation.
